agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2562,17 +2365,6 @@ bool slave_thread::handle_cmdu_monitor_message(
             iface_status_operational_state = false;
         }
 
-        if (slave_state == STATE_OPERATIONAL && notification_in->new_tx_state() == 0 &&
-            notification_in->new_hostap_enabled_state() == 1) {
```

Comment:
We could now merge new_tx_state and new_hostap_enabled_state into a single boolean, but that's possibly a bit of overkill.

Reply:
Basically for operational status to the platform manager, we had defined that the front radio should be enabled, and the should be TXing as well. If we remove the status notification, this could be removed as well.
@tomereli, need your call here.

Reply:
different PR I think.